README.md

```diff
@@ -565,7 +565,7 @@ Table of Contents
    * [codesandbox.io](https://codesandbox.io/) — Online React playground
    * [stackblitz.com](https://stackblitz.com/) — Online VS Code IDE for Angular & React
    * [cacher.io](https://www.cacher.io) - Code snippet organizer with labels and support for 100+ programming languages
-
+   * [Coder](https://coder.com) - Online development environment built around VSCode
```

Comment:
unfortunately there appears no indication of price or what eventual priving or free tier would look like, its just a alpha release.

Great, but I am not going to add something that on release might have no free tiers - so if there's a link I missed on their site that outlines please let me know, we should then update the description here to capture the details of whats available for free

thanks!

Reply:
After looking through their homepage I just noticed that their current pricing model is only described if you are logged in:

https://coder.com/store

![image](https://user-images.githubusercontent.com/23224564/51616538-99be8c00-1f2a-11e9-8237-0ad4738a4fdb.png)

Reply:
hmm, ok, mind updating the description with some version of that? Mention the plans are visible once logged in

Reply:
Hope this is okay :)

Reply:
no update :)

Reply:
My bad ... ^^ 
I only edited it here in the pull request. Should be fine now!